api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -54,6 +54,10 @@
   @Mapping(target = "accessTierShortName", source = "dbWorkspace.cdrVersion.accessTier.shortName")
   Workspace toApiWorkspace(DbWorkspace dbWorkspace);
 
+  @Mapping(
```

Comment:
not sure why this became necessary, but MapStruct stopped converting plain strings.

`WorkspaceMapperTest.testConvertsFirecloudResponseToApiResponse()` saved us by detecting this